src/components/card-link/card-link.module.css

```diff
@@ -30,6 +30,7 @@ style applied to anchor elements.
 		position: absolute;
 		right: 0;
 		top: 0;
+		z-index: -1;
```

Comment:
🗒️ This is a fix for tooltips. They weren't rendering on product badges in tutorial cards because the link's `::before` element was on top of everything and blocking the badges.